### PR TITLE
Adding a KEYS file with public pgp signing keys for Helm releases

### PR DESCRIPTION
**Pull request by mattfarina**
This file contains the keys and instructions for adding KEYS and
importing them. It follows the model Apache uses for a file like
this.

This is part of #6015

### CODE REVIEW

File: KEYS [hunk 1]
```diff
@@ -0,0 +1,148 @@
+This file contains the PGP keys of developers who have signed releases of Helm.
+
+For convenance commands are provided for those who use pgp and gpg.
```
Comment (hickeyma) [inline KEYS]: is it `convenance` or `your convenience` you mean?

File: KEYS [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+For users to import keys:
+       pgp < KEYS
+       gpg --import KEYS
+Developers to add their keys:
```
Comment (hickeyma) [inline KEYS]: How about newline between users and developers?

File: KEYS [hunk 3]
```diff
@@ -0,0 +1,148 @@
+This file contains the PGP keys of developers who have signed releases of Helm.
+
+For convenance commands are provided for those who use pgp and gpg.
+
+For users to import keys:
+       pgp < KEYS
+       gpg --import KEYS
```
Comment (hickeyma) [inline KEYS]: Add an `or`:
       pgp < KEYS
or
       gpg --import KEYS

File: KEYS [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+Developers to add their keys: 
+        pgp -kxa <your name> and append it to this file.
+        (pgpk -ll <your name> && pgpk -xa <your name>) >> this file.
```
Comment (hickeyma) [inline KEYS]: :%s/this file./KEYS/g

File: KEYS [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+        (gpg --list-sigs <your name>
+             && gpg --armor --export <your name>) >> this file.
+
```
Comment (hickeyma) [inline KEYS]: Add `or` between the commands like above for users

File: KEYS [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+pub   rsa4096/0x461449C25E36B98E 2017-11-10 [SC]
```
Comment (hickeyma) [inline KEYS]: Should a separator go above this to clearly mark the commands from the keys info?

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for this @mattfarina. Few small things. Note that I am just eye-balling it and haven't tried out the commands etc.

**Comment by mattfarina** [comment]
@hickeyma thanks for the feedback. Changes made.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks.

**Comment by bacongobbler** [comment]
LGTM! Here's my key if you'd like to add it here. Travelling today so I can't checkout the branch locally in a reasonable amount of time.

```
pub   rsa4096 2018-03-14 [SC]
      967F8AC5E2216F9F4FD270AD92AA783CBAAE8E3B
uid           [ultimate] Matthew Fisher <matt.fisher@microsoft.com>
sig 3        92AA783CBAAE8E3B 2018-03-14  Matthew Fisher <matt.fisher@microsoft.com>
sub   rsa4096 2018-03-14 [E]
sig          92AA783CBAAE8E3B 2018-03-14  Matthew Fisher <matt.fisher@microsoft.com>

-----BEGIN PGP PUBLIC KEY BLOCK-----

mQINBFqpgxYBEAC1+yf/KFw2AQlineurz7Oz8NyYMlx1JnxZvMOFrL6jbZGyyzyy
jBX5Ii++79Wq1T3BL+F/UFhgruQbbzL8SiAc8Q55Ec7z/BVxM7iQPLCnFRqztllx
Ia1D1dZ9aFIw4P92kQgOQGPOgIxFRwEPA0ZX5nbZfL/teNhphW7vHaauk9xEJddm
Pyy3l9xCRIKQVMwuCaLeH0ZZpBllddwuRV4ptlQ30MpOnaalQda9/j3VhNFEX8Nj
nu8GHn+f4Lzy6XmhHb++JB3AIo5ZfwaUS2xMrnObtvmGHR3+uP/kblh9MzZlmL4T
ldclyGaV7z9Z/xGwnX/+r7xna/fr3mey3GXm29BOP2sUBBQCba05X5nYUd2TjWsZ
OZtE6sLuzUzeOTLEDu28IJoiaYnLKDNzDmuVM26xAYVWXUdCGgn+1rAp0t5OGgHm
qTexvPmckgp3yw+tcPUkR6nh0ft7pmeoK53AQHMt6fk7plZCTuu5UvxZE/oDzt4X
w9+vSTD5GzsNGrTYLTYUSL0muK+iM/uuJtFNJUREOucXfmWxulUsxwOB0st7hnLs
4JmFSr3av1en1WqqdiXswOrdK2msTm4J2+fsOU1jnyF//RJmj+1KPpRDCBTzpAFS
SzE/rRaLZBVE8k2vT0L6yBXvGJ2ONK9TkGT5fnyXu8zDu1d2Koj0c+6m9wARAQAB
tCpNYXR0aGV3IEZpc2hlciA8bWF0dC5maXNoZXJAbWljcm9zb2Z0LmNvbT6JAk4E
EwEIADgWIQSWf4rF4iFvn0/ScK2Sqng8uq6OOwUCWqmDFgIbAwULCQgHAgYVCgkI
CwIEFgIDAQIeAQIXgAAKCRCSqng8uq6OOyTsD/979LDS7ONHIHNoRf7Uud40To0S
/domtZM0rXUCBdbe5R4/xah0HvM1u8aN4OC6U7i0LCXSmEOZxQLKxKBWfX4/d6k7
lBwuQBSlcM6cM6nDfPInT0C3o8caP8lOGeNAdOkMxrqiEO4gHNP5BvWCV+jQSU5X
uvGhKNTMcpaf+DqZAFbR6zpdL7t5JCK0B0RRhFfaGWb19t3REukI5OF5M5SN7EtQ
XWK/1fyzsltrjTSXgMWuxtJjBchltjme/S3XpHeeoSCm1WWh3a140tCC662ydU1u
EZIlUrn8dfMpH0BY6bb0/4dhHvCJ3bw+zZoCzFJM/LksjP5i+Q4mUOD8PvFWh5aS
46F827YiMdqD/eDMr1QRe66fPw5EtWTHgnf3PX+NmN8lgn2o280AkRXqkrCgl580
B+lFwZ6hfan2F8RIHXNbF+9Zvc7Nh8bG8s4I8s6uiufmsmOuFdp47J4//q1W0HcU
0fqajDnEhExtGkgwIsum1Ndwq2sWZT/ko7PYyC3J6mbr/MXTvd2TxtnMgG6kpyPv
p3HlDaBw1aO5vO5mji4RTsoZi12MITIyvPsFWh0WtXkJLNaJ30bFSEx5fiJILxu0
bBoBK0LUhB1Q+8G3Kea3+q3MuOQFnFfjPlMH6q84jpU5Lv5BaW17IeZ2kIfVYrcG
vBvtZ5VHDzY4EhGmlbkCDQRaqYMWARAA3wYv6jbE1PjXwIUWSSO9zxQLBKg7Cn7d
g+wwKx+N5DHjSdQBous6DGwN/wEZfXJOn14S9Yg4p4owmiyJDn0oqJ0BLdsMELoO
imCIZ+zn3AjCWdk2b0oCOhyTwhaVhVgi8yMQruMSUG9/3lkVoFae/GMC32nmE2A0
BOnj9fVIhIrDKt9OSeTXXRNVaRvNFo9ry8S1hDxgfQ2unD6J0mMPhLH2O7CRZDFW
FyH09E/rhrIDvI3Z7mZw2ufGKR0YEu7fJ0BBBSbIqUOMsUnQNWomb2j/QZyYmhTS
Hg9YRB807H3b+5GuZim+DSUk5DQV2IENEg9LDYvhDftE5COYB3tZUnvEpOvNybBl
URxD8Kgqlb3j93l2FcD1QrIGW5VCmkkuD612ZG+NjMq0ZXlQjv6gxAYir8GTKkWt
tS1OatDm6qe6xEFypT6nlvxOYFxLeFkVVGt4H4QW6+MXvnwMofL0G6fOhRvdlq3R
US9n3WqzTpCwfvJs2lhYi+c3/2nwCx5G42OT9Ix0UFkYwxhGk6PRleKOMsw28PFr
a8DVjyKGOVn+9auVhPXYQcN0sZqFl8LBDkUtaniiRD4WKH91aKYgmX1qo8sJZMhx
t/ZoHOfoHDEEa+kLqfsWu3htyTP1gleCAA8kDcRiy1v/G8v3+p2ioI6q1qegigbr
AqTHcWNOltcAEQEAAYkERAQYAQoADwUCWqmDFgUJAAAAAAIbDAIpCRCSqng8uq6O
O8FdIAQZAQoABgUCWqmDFgAKCRDpVu6v5D8W5zYaD/9a6ZMCmoEHJb/X3UrsRA2A
iFlsDK8lkGgv8ulU1U1U1fDFL3/OgGtO6gIZeVexNzVRyLRZsuul55m870HGT0MO
3AaKN8An0uhc+uhIRND7cs6E9iMxNGXBjsJvbvH3yVVbtphKFsjy21EKUJwA8BKn
dw6ZdO95+AsflcbntLxUjoG1USqBagPskUULSTF8wrRReFHufXVy2rW/MiJpoSrd
9CpIumYSm7Gf8ELRMduTxYwEJrCzwGrqbN5rrt+doEwkfWNgHTnSFTKasuXv8P7M
I8GgtBcXaYpTVzBinrQHifyaPxhCjzfCqBsqaZDLw8qicqR8KhM6YYH1YWWg5brC
FpC/BuhasFM8R0am9iB7D7xCJ0slRoBKCmSOG4MoWx6RS5lyJTnwSj+ZQwnQQ63L
xcIPUHrFmmiAgKgqhI8AQpVQ4S+YPWaWNUlHYq3hrce4++7xRrCHekMz2rkgD0w8
PkwAYWSKbsFGIzhXxSCuxtHRDIhOIx5tF1e8/pnE9YBdghPIVWboGYBBfkGO0PXw
pO5v8CKTWMvvZiGFficf86vY8A8m0IgAgbc03DuaeDcEImAhYqJ4/Q9O3PdSy+R9
/1peEreAqv+4w5Oxk53/TTDvmtYqk7GK7wLDm5VrimOEcqsZZTCFBA2mAi/c00UQ
9zDJJpm6v54dplKAEulqogWbD/9hoJdNPOnmtuE9hLrGQ2FrBlSWDqksA43jXuap
9QHIHBjbDTOdpb8PCYn/Q4emekaszGiKOrAmaujHP8lKE2OiGWj4OwvKb8pxvxDy
cKbl0ophTfvwp4rZyvGGKYxepGAf7wjyAJPo8M1Mu1B4UN9EvKYSMphh7qommoS2
WUPdDyZHFi0eO/IIe3SKj3bzDhSbZ0+vfCK1bnIGfgFg7LkFjmcfHlZiKQh6triO
h7fTKFzKQbR/3r0pi1zkcqLuMHqNoIsO0dKKytvXYU09+fTYYYvx3k3r2Tk35ogc
//Dh/W9IW/qzSquSkrvYknr1rAs4+YLsQBRMaVZA0Gjl/esn9oWvYoTB17kDOxnR
X8ijlJqtMy6bBiPMMh5trp6pvzPPcnMrL0jAUgbbaq6FcGD7CoqfNUrB3zZBMLDN
SB2/th4Ydtimabg0eD3DJ09tr1B04VcQ6qUpRI5iCEhv0fgnXEPjuPdB00AHwWF/
5DJNMRxu+fWk7rhjsG4O3Rq2S2XC55hQTZhSs5YwwXcfPfZSm21dIn7edwzS9X3m
2+ECe8SXnsnEg060dHzSXCn1LA9XW0gNg69TW30FDmqeugTT2aIHuxpHO27afRUB
AydmIjj/uzPvvDZdDcJ0yno5pN+PFF+JHsY1Nn9Ci4jeQO82tszpsxia9vcKBMal
BttQow==
=MQfj
-----END PGP PUBLIC KEY BLOCK-----
```

**Review by bacongobbler — APPROVED** [review]

**Comment by mattfarina** [comment]
@bacongobbler I didn't add your public key because I see a potential issue with this. Let's connect in sync on it.

**Pull request merged**